src/cli.rs

```diff
@@ -1,7 +1,27 @@
-use clap::Parser;
+use clap::{Parser, Subcommand};
 
 use serde::Deserialize;
 
+/// What to do with the selected container.
+#[derive(Subcommand, Deserialize, Debug, Clone, Copy)]
+pub enum Commands {
+    /// Focus the container
+    #[command(about = "Focus selected window")]
+    Focus,
+
+    /// Swap current window with selected window
+    #[command(about = "Swap focused window with selected window")]
+    Swap {
+        /// Focus the selected container after swapping
+        #[arg(long)]
+        focus: bool,
```

Comment:
I feel like the naming here is a bit confusing---is there some way that we could emphasize that `focus == true` means that the originally focused container gets unfocused (I can't think of a better way to word that right now, sorry).

Reply:
I don't think we should mention `true` here because it shows up in the `--help` output and that might be confusing.  This is what it looks like now:

```
Swap focused window with the selected window

Usage: sway-easyfocus swap [OPTIONS]

Options:
      --focus  Also focus the selected window after swapping
  -h, --help   Print help
```

Reply:
Oh I see. Good call, the message looks better in context.